notebooks/olmocr-pdf-vlm/olmocr-pdf-vlm.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.    main_image = Image.open(BytesIO(base64.b64decode(image_base64)))
</pre><p>it is also good to visualaze input image before generation here</p><p> </p><p>you can do it in the same cell just adding line:</p><p> </p><p>display(main_image)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Image display moved here</p>